components/org.wso2.carbon.identity.entitlement.proxy/src/main/org/wso2/carbon/identity/entitlement/proxy/soap/authenticationAdmin/SOAPEntitlementServiceClient.java

```diff
@@ -57,19 +61,38 @@
     private String userName;
     private String password;
     private boolean reuseSession = true;
+    private String authorizedCookie;
+    Authenticator authenticator;
 
-    public SOAPEntitlementServiceClient(String serverUrl, String username, String password, boolean reuseSession) {
+    public SOAPEntitlementServiceClient(String serverUrl, String username, String password, boolean reuseSession)
+            throws EntitlementProxyException {
```

Comment:
Hi @lahiru823,

Isn't this an API change ?. Was this previously discussed with the stakeholders ?